src/www/templates/group/groupDetails.html

```diff
@@ -1,10 +1,28 @@
 <!--TODO Abstract from container-->
 <div class="container">
-    <h1>{{group.displayName}}</h1>
-    <h4>Group description</h4>
-    <p>{{group.description}}</p>
-    <h4>Contact details</h4>
-    <p>{{group.email}}</p>
-    <h4>Members</h4>
-    <p ng-repeat="member in group.members">{{member.displayName}} ({{member.user_group.func}})</p>
+    <h1>
+        <a class="btn btn-link" href="javascript:history.back()">
+            <i class="fa fa-chevron-left" aria-hidden="true"></i>
+        </a>
+        {{group.displayName}}</h1>
+    <div class="row HA-col-container">
+        <div class="col-sm-6 col-md-6">
+            <h2>{{group.fullName}}</h2>
+            <p>{{group.description}}</p>
+            <i>Since {{group.createdAt | date: 'dd MMM, yyyy'}}</i>
```

Comment:
This doesn't work for me; I think the type `createdAt` hasn't been defined yet.

Reply:
Yeah, this is a back-end fix that was lost when I had to do a hard reset. I will create a matching committee branch on the backend for this fix.